management/server/types/group.go

```diff
@@ -26,14 +26,41 @@ type Group struct {
 	Issued string
 
 	// Peers list of the group
-	Peers []string `gorm:"serializer:json"`
+	Peers      []string    `gorm:"-"`
+	GroupPeers []GroupPeer `gorm:"foreignKey:GroupID;references:id;constraint:OnDelete:CASCADE;"`
```

Comment:
can these be documented as read only, and that `[Create/Update]Group` will ignore any changes to these, instead `[Add/Remove]Peer[To/From]Group` should be used instead?